github

```diff
@@ -0,0 +1,3 @@
+f = loadfile('github.rom/programs/github')
+setfenv(f, getfenv())
+f(...)
```

Comment:
I think its more readable this way imo

On Thu, Aug 25, 2016 at 2:43 AM Eric Wieser <notifications@github.com>
wrote:

> In github
> <https://github.com/eric-wieser/computercraft-github/pull/4#discussion_r76194751>
> :
>
> > @@ -0,0 +1,3 @@
> > +f = loadfile('github.rom/programs/github')
> > +setfenv(f, getfenv())
> > +f(...)
>
> I guess there's no advantage to having it on one line
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/eric-wieser/computercraft-github/pull/4/files/783fcd1b03426075530e74999577024845c3eed5#r76194751>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA41FptO0gA7dH6E0CNAAGjYZZrngKjuks5qjUeFgaJpZM4JoHt0>
> .
>